docs/faq.yaml

```diff
@@ -5,75 +5,97 @@
     databases. But we would like to have them use the same login dialog for our portal.
     How can I achieve that with Ory Kratos?
   a: You can deploy Ory Kratos two times, and use the same login/registration endpoints
-    if you like. You may need to tell you login/registration ui which Kratos it is
-    supposed to talk to.
+    if you like. You may need to tell your login/registration UI which Kratos it is
+    supposed to talk to. The instances are cheap to deploy and the databases are completely
+    isolated from each other.  For example you can have something like `/login/customer`
+    and `login/employee`.
+- tags:
+  - kratos
+  - ecosystem
+  - bug
+  q: Will you audit the code and/or provide bug bounties?
+  a: Yes, when we are near stable candidate we will audit the code and provide bug
+    bounties.
 - tags:
   - kratos
   - user
   - validation
   - traits
   q: What is the correct flow to sanitize the username and make sure it satisfies
     a specific regex (e.g. only alphanumeric characters)?
-  a: 'In the identity json schema using https://json-schema.org/understanding-json-schema/reference/string.html#regular-expressions.
+  a: In the identity json schema using [Regular Expressions](https://json-schema.org/understanding-json-schema/reference/string.html#regular-expressions).
     To sanitize usernames coming from third-party OIDC providers like Google or GitHub
-    you write jsonnet code: https://jsonnet.org.'
+    you write [jsonnet code](https://jsonnet.org).
+- tags:
+  - kratos
+  - registration
+  - cors
+  q: 'Hey guys, I am currently trying to get the Kratos registration flow working
+    on a react app that I am creating. I am having CORS issues with the completeSelfServiceRegistrationFlowWithPasswordMethod
+    call when using the @oryd/kratos-client JS package. '
+  a: You are using AJAX to submit the form. This is not supported right now. You need
+    to submit a regular HTML form. Do not use AJAX. Do not use fetch. Let the user
+    click the submit button and do not do anything. AJAX will be better supported
+    starting Kratos 0.6.  s
```

Comment:
```suggestion
    starting Kratos v0.6
```

Reply:
slipped on the keyboard there apparently...